CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Lotus changelog
 
+# 1.28.2 / 2024-08-15
+
+This is a Lotus patch release v1.28.2 for Node operators and Storage Providers.
+
+For node operators, this patch release fixes an issue where excessive bandwidth usage (issue #12381) was caused by a routing loop in pubsub, where small "manifest" messages were cycling repeatedly around the network due to an ineffective routing loop prevention mechanism.
+
+For storage providers this patch release fixes pledge issues, users have been encountering. This update addresses existing issues, including the too-small pledge in snap and the lack of DDO-awareness in PoRep Commit.
```

Comment:
```suggestion
For storage providers, this patch release fixes pledge issues users have been encountering. This update addresses existing issues, including the too-small pledge in snap and the lack of DDO-awareness in PoRep Commit.
```

Also, do we have an issue we can link to?

Reply:
seems like the reports didnt get from slack to github